### ISSUE TITLE
release package on pypi

### ISSUE THREAD

**Issue by mpacer** [jupyterlab/jupyterlab-latex#22]
@ian-r-rose  do you think this is ready to be released as an early version on pypi so that others can more easily test it? I was thinking of bringing this up at today's jupyter dev meeting (2017_W50_2) and I realised I'd need to ask people listening to clone the repo to get it to work. 

If it's not ready what should we work on to get it ready?

**Comment by ian-r-rose** [comment]
I wanted to play with it using a larger, more complex .tex document, to see if there are any issues. I saw some async loading issues with PDF.js previously, but it is possible that we squashed those in our bug-fixing session.

I was also thinking that we should reduce the ranking of the mimerenderer plugin so that it does not override the native PDF viewer by default.

We would also need to release it on npm.

**Comment by ian-r-rose** [comment]
I wanted to play with it using a larger, more complex .tex document, to see if there are any issues. I saw some async loading issues with PDF.js previously, but it is possible that we squashed those in our bug-fixing session.

I was also thinking that we should reduce the ranking of the mimerenderer plugin so that it does not override the native PDF viewer by default.

We would also need to release it on npm.
